package.json

```diff
@@ -27,31 +29,32 @@
   },
   "devDependencies": {
     "@babel/core": "^7.14.6",
+    "@babel/plugin-transform-typescript": "^7.18.8",
     "@babel/preset-env": "^7.14.7",
     "@babel/preset-react": "^7.14.5",
     "@testing-library/react": "^12.0.0",
+    "@types/jest": "^28.1.5",
+    "@types/video.js": "^7.3.42",
```

Comment:
since video.js is a peer dependency, is it possible to move @types/video.js to peer dependencies as well to not couple this package with a specific video.js version?

Reply:
Hmm, maybe... But it looks like you already have video.js as a straight dependency, so I don't think adding the types would lock us into one version of video.js than we already are...

https://github.com/jimmycallin/react-hook-videojs/pull/148/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R54

Reply:
`video.js` is only used as a dependency for the tests, and is not used in the actual output.

I was afraid that the types from `@types/video.js` was bundled in the `dist/index.d.ts`, but looking at the output it doesn't seem to be, but will instead be imported by the user, if i understand `dist/index.d.ts` correctly. But this will require the user to also install `@types/video.js` along with `video.js`, and therefore we should include `@types/video,js` as peerDependencym, is that correct?

Reply:
@jimmycallin Hmm, I see what you mean. I think they'd only need to install @types/video.js if they themselves are using TypeScript. If they aren't using TypeScript they wouldn't need it. So I'm not sure we would include it as a peer dependency for that. 🤔

Reply:
Looks like it will be fallback as `any` if @types/video.js isn't installed, which makes sense to me.